nrf_ble_dfu_controller.py

```diff
@@ -198,10 +204,24 @@ def _dfu_wait_for_notify(self):
                 #
                 self.ble_conn.sendline('')
                 string = self.ble_conn.before
-                if '[   ]' in string:
+                if b'[   ]' in string:
                     print('Connection lost! ')
                     raise Exception('Connection Lost')
-                return None
+                 else:
```

Comment:
This else comment has an extra space of indent.
```suggestion
                else:
```